app/models/tag.rb

```diff
@@ -0,0 +1,7 @@
+class Tag < ApplicationRecord
+  # rubocop:disable Rails/HasManyOrHasOneDependent
+  has_many :movies_tags
+  has_many :movies, through: :movies_tags, dependent: :destroy
+  # rubocop:enable Rails/HasManyOrHasOneDependent
+  validates :name, length: { minimum: 2, maximum: 30 }
```

Comment:
Can I create multiple tags with the same name?

Reply:
when you fix it, please write a test to fix this case)
also look at other db tables with your team, maybe there is the same issues there?

Reply:
we decided to fix it in another PR, but our mistake was that we threw the PR without taking into account your last review

Reply:
yes of course you can fix it in this PR because you are creating a new table)
This is much more productive than the other PR. Since the data will conditionally go to prod and you will no longer be able to change the index so easily) you will have to delete it and create new ones, this operation can take a decent amount of time if there are a lot of records in the table)